site/_docs/installation.md

```diff
@@ -31,16 +32,18 @@ earlier, for CoffeeScript support).
   </p>
 </div>
 
-## Install with RubyGems
+## Install with Bundler and RubyGems
 
-The best way to install Jekyll is via
+The best way to install Jekyll is using a Gemfile via Bundler
 [RubyGems](http://rubygems.org/pages/download). At the terminal prompt,
```

Comment:
The standard way to install Jekyll on a new system is still `gem install jekyll`
But if for some reason, one decides to try Jekyll out using an existing repo, and if that repo was created with Jekyll 3.2, then, the best way for him to use bundler. He'll be able to install jekyll with all its dependencies and the repo's theme, with one command `bundle install`
To get the repo-site served just after installation with minimal time, from the repo root, he just needs to enter 
```
$ bundle && bundle exec jekyll s
```